src/java.base/share/classes/java/lang/System.java

```diff
@@ -1613,10 +1359,6 @@ public void log(Level level, ResourceBundle bundle, String format,
      * should not attempt to configure loggers by themselves, as that
      * would make them dependent from a specific implementation of the
      * {@code LoggerFinder} service.
-     * <p>
-     * In addition, when a security manager is present, loggers provided to
-     * system classes should not be directly configurable through the logging
-     * backend without requiring permissions.
      * <br>
      * It is the responsibility of the provider of
      * the concrete {@code LoggerFinder} implementation to ensure that
```

Comment:
This is still a part of the paragraph related to the security manager.

Reply:
Right - this paragraph - lines 1620-1625 (old file) / 1362-1367 (new file) should be removed too. Thanks for spotting that.

Reply:
Removed in jep486 branch in sandbox so will get picked up when PR is refreshed.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/ab586f6619ca5f7e213876219abf61a36155735c